smoke-test/config.yaml

```diff
@@ -0,0 +1,33 @@
+General:
+  ConfigurationVersion: 2
+  MinRefineryVersion: v2.0
+  ConfigReloadInterval: 50s
+
+Network:
+  HoneycombAPI: https://api-dogfood.honeycomb.io
+
+Logger:
+  Type: stdout
+  Level: info
+
+LegacyMetrics:
+  Enabled: true
+  Dataset: refinery_metrics
+  APIHost: https://api-dogfood.honeycomb.io
```

Comment:
I'm not sure if we want to send this local refinery's metrics into this dataset. It could be confusing since that's where the production refinery sends its metrics

Reply:
These *are* the same EXCEPT that which team and environment receive the telemetry depends on the API key provided. There is no default key in this config, so a fresh clone of the project running this setup won't send anywhere. 

I am happy with any config we put in here.